packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php

```diff
@@ -1471,35 +1471,23 @@ public function testGetUpdateWithPkCommandNullManipulationWithTimestamp(): void
-            'UPDATE "import_export_test_schema"."import_export_test_test" AS "dest" SET "col1" = "src"."col1", "col2" = "src"."col2", "_timestamp" = \'2020-01-01 01:01:01\' FROM "import_export_test_schema"."__temp_stagingTable" AS "src" WHERE "dest"."col1" = "src"."col1"  AND (not(EQUAL_NULL("dest"."col1", "src"."col1")) OR not(EQUAL_NULL("dest"."col2", "src"."col2")))',
-        ];
-        yield 'import:row-change:IS_NULL' => [
-            'import:row-change:IS_NULL',
-            // phpcs:ignore
-            'UPDATE "import_export_test_schema"."import_export_test_test" AS "dest" SET "col1" = "src"."col1", "col2" = "src"."col2", "_timestamp" = \'2020-01-01 01:01:01\' FROM "import_export_test_schema"."__temp_stagingTable" AS "src" WHERE "dest"."col1" = "src"."col1"  AND (("dest"."col1" != "src"."col1" OR ("dest"."col1" IS NULL OR "src"."col1" IS NULL AND ("dest"."col1" IS NULL AND "src"."col1" IS NULL))) OR ("dest"."col2" != "src"."col2" OR ("dest"."col2" IS NULL OR "src"."col2" IS NULL AND ("dest"."col2" IS NULL AND "src"."col2" IS NULL))))',
-        ];
-        yield 'import:row-change:HASH' => [
-            'import:row-change:HASH',
-            // phpcs:ignore
-            'UPDATE "import_export_test_schema"."import_export_test_test" AS "dest" SET "col1" = "src"."col1", "col2" = "src"."col2", "_timestamp" = \'2020-01-01 01:01:01\' FROM "import_export_test_schema"."__temp_stagingTable" AS "src" WHERE "dest"."col1" = "src"."col1"  AND (HASH("dest"."col1", "dest"."col2") != HASH("src"."col1", "src"."col2"))',
-        ];
-        yield 'import:row-change:sub:COALESCE' => [
-            'import:row-change:sub:COALESCE',
-            // phpcs:ignore
-            'UPDATE "import_export_test_schema"."import_export_test_test" AS "dest" SET "col1" = "src"."col1", "col2" = "src"."col2", "_timestamp" = \'2020-01-01 01:01:01\' FROM "import_export_test_schema"."__temp_stagingTable" AS "src" WHERE "dest"."col1" = "src"."col1"  AND (COALESCE("dest"."col1", \'KBC_$#\') != COALESCE("src"."col1", \'KBC_$#\') OR COALESCE("dest"."col2", \'KBC_$#\') != COALESCE("src"."col2", \'KBC_$#\'))',
+            'UPDATE "import_export_test_schema"."import_export_test_test" AS "dest" SET "col1" = "src"."col1", "col2" = "src"."col2", "_timestamp" = \'2020-01-01 01:01:01\' FROM "import_export_test_schema"."__temp_stagingTable" AS "src" WHERE "dest"."col1" = "src"."col1" ',
+            true,
         ];
-        yield 'import:row-change:DISTINCT' => [
-            'import:row-change:DISTINCT',
+        yield 'new-native-types' => [
+            'new-native-types',
             // phpcs:ignore
             'UPDATE "import_export_test_schema"."import_export_test_test" AS "dest" SET "col1" = "src"."col1", "col2" = "src"."col2", "_timestamp" = \'2020-01-01 01:01:01\' FROM "import_export_test_schema"."__temp_stagingTable" AS "src" WHERE "dest"."col1" = "src"."col1"  AND ("dest"."col1" IS DISTINCT FROM "src"."col1" OR "dest"."col2" IS DISTINCT FROM "src"."col2")',
```

Comment:
Nejde tady použít `IS NOT DISTINCT FROM`?

Reply:
Myslim namisto `"dest"."col1" = "src"."col1"` pouzit `"dest"."col1" IS NOT DISTINCT FROM "src"."col1"`